src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -70,6 +70,7 @@ export const DesktopDropdown = <T extends OptionType>({
           aria-haspopup="menu"
           role="button"
           ref={initialFocus}
+          aria-label="dropdown"
```

Comment:
Would we want `aria-label` to refer to the dropdown as a whole or the specific options label that's currently in focus? `aria-label={currentOption ? currentOption.label : 'dropdown'}`